code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -698,5 +698,5 @@
 	..()
 	if(ishuman(M))
 		if(M && M.stat == DEAD)
-			M.stat = 0
-			M.visible_message("<span class='danger'>\The [M] shudders violently!!</span>")
\ No newline at end of file
+			M.revive()
```

Comment:
Somehow I don't think a full organ replace fits with the theme of this.

Reply:
Turns out, neither this seems to be working.

Reply:
Should be fixed now.